interface/billing/sl_eob_invoice.php

```diff
@@ -159,34 +159,35 @@ function validate(f) {
                     }
                 }
                 if ((cPay !== 0) && isNaN(parseFloat(f[pfx + '[pay]'].value))) {
-                    dlgopen('', '', 675, 250, '', 'Warning', {
+                    dlgopen('', '', 675, 250, '', '<div class="text-danger">Warning</div>', {
                         type: 'Alert',
-                        html: '<p><?php echo xlj('Payment value for code ') ?> + code + <?php echo xlj(' is not a number') ?></p>',
+                        html: '<p>Payment value for code ' + code + ' is not a number</p>',
                     });
                     return false;
                 }
                 if ((cAdjust !== 0) && isNaN(parseFloat(f[pfx + '[adj]'].value))) {
-                    dlgopen('', '', 675, 250, '', 'Warning', {
+                    alert('OK')
+                    dlgopen('', '', 675, 250, '', '<div class="text-danger">Warning</div>', {
                         type: 'Alert',
-                        html: '<p><?php echo xlj('Adjustment value for code ') ?> + code + <?php echo xlj(' is not a number') ?></p>',
+                        html: '<p>Adjustment value for code ' + code + ' is not a number</p>',
                     });
                     return false;
                 }
                 if ((cAdjust !== 0) && !f[pfx + '[reason]'].value) {
```

Comment:
> if the adjDisable is turned on should not have any popup alerting to select an adjustment code
hi @stu01509 , this is what needs to have an extra condition to handle this since the global is meant to suppress these warnings and allow the user to save the adjustment without a reason code, it basically allows you to change the value of the radio boxes and save without being annoyed by the warning.

The warning box size could probably be reduced as well
![Screenshot from 2020-03-25 16-23-11](https://user-images.githubusercontent.com/5618372/77581972-03328580-6eb5-11ea-8d4c-283f2ddc87ff.png)

Reply:
nice sizing, and the if should be
```
                if ((cAdjust !== 0) && !f[pfx + '[reason]'].value  && !adjDisable) {

Reply:
@stephenwaite 
Do I need to add in all dlopen if condition?

Reply:
no thank you